src/packets/IS_OBH.ts

```diff
@@ -25,7 +24,8 @@ export class IS_OBH extends Packet {
   @word() Time = 0;
 
   /** Contact object */
-  @byteArray(8) C: CarContOBJ = new CarContOBJ();
+  //@byteArray(8) C: CarContOBJ = new CarContOBJ();
```

Comment:
Now that the `@struct` is working, you can remove the commented out part

Reply:
removed.